### ISSUE TITLE
Self-development

### ISSUE THREAD

**Issue by kgorana** [bcgov/parks-data-register#497]
## Description
Listing the tasks I want to do in the next couple days that are not my job duties

## Acceptance Criteria
- [x] Update **all** PDP Goals
- [ ] Policy Essentials e-learning (1 hr)
- [ ] Agile Fundamentals e-learning (2 hr)
- [ ] ~~SM/PO tasks by helping Diana out (if she needs help)~~
- [x] Agile CoP volunteering task
- [ ] https://digital.gov.bc.ca/topics/agile/
- [ ] https://digital.gov.bc.ca/cloud/saas/intro/
- [ ] Recommendations List from August 2023 emails
- [x] PDP Goals revision + follow up meeting with Jess
- [x] clean inbox
- [x] DM CoP - terms of acknowledgement co-facilitation
- [x] coaching session and notes

## Blocked by
- #474 
- #123 
- #484 

##Definition of Ready
- [x] Acceptance criteria are included
- [x] Design / Solution is accepted by Product Owner (if applicable)
- [x] Dependencies are identified (technical, business, regulatory/policy)
- [x] Story has been estimated (under 13 pts)

## Definition of Done
**In progress:**
  - [x] Acceptance criteria are tested
  - [x] Work is traceable in GitHub

**PO Review:**
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

## Notes
-

**Comment by kgorana** [comment]
Projects/tasks related to my area of interest and business needs ideas summarized so I don't forget:
- sit with Winnie in user experience research interviews
- DUP June 2024 research
- PIA/STRA

**Comment by kgorana** [comment]
working on Agile CoP tasks right now; will write the outcomes here with time:
- helped facilitate a workshop with community members to draft mission and vision of Agile CoP `past`
- going to help with affinity mapping of the workshop outcomes `future`
- assisting with organization of materials on SharePoint `ongoing`
- will develop a roadmap next week (week of July 8, 2024) `future`

**Comment by LindsayMacfarlane** [comment]
@Dianadec - this ticket appears to be Kavi specific - assuming we can close or delete. Thoughts?

**Comment by LindsayMacfarlane** [comment]
Confirmed with Diana and Jess that this ticket is no longer needed. Closing.
